.typos.toml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: Copyright (C) 2025 Opal Health Informatics Group at the Research Institute of the McGill University Health Centre <john.kildea@mcgill.ca>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+[default.extend-words]
+muhc = "muhc"
```

Comment:
`muhc` only appears in `package.json`, but we might want to remove that.

Reply:
Good point, I removed it.

Reply:
Thanks!